### ISSUE TITLE
DNS server floods logs when filtering out unhealthy nodes

### ISSUE THREAD

**Issue by pbitty** [hashicorp/consul#1369]
[This line](https://github.com/hashicorp/consul/blob/921602b565b04d59ac634d53542d05dfd93a6ca3/command/agent/dns.go#L549) in dns.go logs unhealthy nodes that are filtered out on every single DNS query. eg:
```
2015/10/29 21:25:08 [WARN] dns: node 'c1a5e456b75a' failing health check '_service_maintenance:www: Service Maintenance Mode', dropping from service 'www'
```

In a high traffic system, the log file can grow very quickly and fill the disk.  I can hide these messages by setting the log level to ERR, but I'm thinking these messages should be at the DEBUG level.

My rationale is that it is normal behavior for consul to exclude nodes from DNS requests if they don't appear as healthy.  If the goal is to log the unhealthy state of a node under a service, could not just log the state change when it's received from the cluster?

**Comment by ryanuber** [comment]
Makes sense to me to downgrade the severity of these to DEBUG, since that's exactly what you would most likely use this log information for anyways. Marking as an enhancement, thanks!

**Comment by slackpad** [comment]
I was in this code for prepared queries and I went ahead and deleted this log message altogether since it's spammy even at debug levels, and you can query the health endpoint to see what DNS is seeing. Once https://github.com/hashicorp/consul/pull/1389 is merged this should get fixed (will go out in 0.6).

**Comment by slackpad** [comment]
Closed via https://github.com/hashicorp/consul/issues/1369.
